### PR TITLE
test(Proxy): skip Proxy.test.js

### PR DESCRIPTION
**Pull request by hiroppy**
This test is unstable when running on Node v11 on MacOS.

- [ ] This is a **bugfix**
- [ ] This is a **feature**
- [ ] This is a **code refactor**
- [x] This is a **test update**
- [ ] This is a **docs update**
- [ ] This is a **metadata update**

### For Bugs and Features; did you add new tests?

N/A

### Motivation / Use-Case

see https://github.com/webpack/webpack-dev-server/pull/1633#issuecomment-463982581

### Breaking Changes

no 

### Additional Info

`HistoryApiFallback.test.js` seems to be good when we add skipping code to `Proxy.test.js`.

### CODE REVIEW

File: test/Proxy.test.js [hunk 1]
```diff
@@ -62,6 +63,11 @@ function startProxyServers() {
 }
 
 describe('Proxy', () => {
+  // issue: https://github.com/webpack/webpack-dev-server/pull/1633#issuecomment-463938675
+  if (shouldSkipTestSuite()) {
+    return;
+  }
```
Comment (evilebottnawi) [inline test/Proxy.test.js]: All tests failed? Maybe we can skip only one test?

Comment (hiroppy) [reply test/Proxy.test.js]: As far as I confirm, running two tests seems to fail.

Comment (evilebottnawi) [reply test/Proxy.test.js]: @hiroppy can we ship only their, thanks?

Comment (hiroppy) [reply test/Proxy.test.js]: The first test(anyone) will succeed but subsequent tests will fail. 
Probably postprocessing would be a problem.

Once we merge this PR and then I think that it is better to investigate the cause separately.(maybe difficult)

Comment (evilebottnawi) [reply test/Proxy.test.js]: :+1: Let's do it

**Pull request merged**